### ISSUE TITLE
Sparkline doesn't reflect trend very well

### ISSUE THREAD

**Issue by ryantibs** [cmu-delphi/www-covidcast#1111]
It seems to me like our dashboard's sparklines sometimes don't reflect the underlying trends very well.  See the example two screenshots from mobile below (sorry for the full-width, it looks like GitHub does that by default), though to be clear I don't think it's a mobile/desktop issue. 

Covid cases appear to be blowing up on the first time series plot, but the sparkline doesn't really reflect that in the indicator table. This isn't a very specific request (sorry), but is there anything we can to get the sparklines to reflect the trend better?

![IMG_0977](https://user-images.githubusercontent.com/732541/148086660-b2b1bee8-6525-4407-be55-151520c8b4f5.PNG)
![IMG_0978](https://user-images.githubusercontent.com/732541/148086663-8303a431-2e8a-4799-98c0-a08d54a00167.PNG)

**Comment by ryantibs** [comment]
To be clear, this is low priority compared to the other v3.1 items.  If we can't find a good solution with relatively little effort, then it's fine: we can just move on without it.

**Comment by sgratzl** [comment]
so far the sparklines have no fixed y-axis limits and adapt to the data going from the minimal value to the maximal value within the four week period. It is the most generic way that works for all signals and changes without manual intervention.

**Comment by ryantibs** [comment]
Right, I understand that's generally the way it works. I was just hoping there might be some generic tweaks to help it show trends better. For example, tweak the aspect ratio so that it's a bit taller than it is now. But as I said, this is pretty low priority, so if there's not a relatively easy fix, then we should move on.

**Comment by ryantibs** [comment]
Removing this from consideration for v3.1 since I don't get the impression that Sam sees it as an easy fix.
